### PR TITLE
Ros

### PR DESCRIPTION
**Pull request by rosrossito**
Make naming consistent #499

### CODE REVIEW

File: javaparser-core/src/main/java/com/github/javaparser/CommentsInserter.java [hunk 1]
```diff
@@ -76,19 +76,19 @@ private void insertCommentsInCu(CompilationUnit cu,
         // so I could use some heuristics in these cases to distinguish the two
         // cases
 
-        List<Comment> comments = commentsCollection.getAll();
-        PositionUtils.sortByBeginPosition(comments);
-        List<Node> children = cu.getChildrenNodes();
+        List<Comment> commentsList = commentsCollection.getAll();
+        PositionUtils.sortByBeginPosition(commentsList);
```
Comment (matozoid) [inline javaparser-core/src/main/java/com/github/javaparser/CommentsInserter.java:80]: The idea was to fix naming in the nodes, not outside it.

File: javaparser-core/src/main/java/com/github/javaparser/CommentsInserter.java [hunk 2]
```diff
@@ -76,19 +76,19 @@ private void insertCommentsInCu(CompilationUnit cu,
...
+        List<Node> children = cu.getChildrenNodesList();
```
Comment (matozoid) [inline javaparser-core/src/main/java/com/github/javaparser/CommentsInserter.java:81]: This was changed weeks ago. How old is your branch?!

File: javaparser-core/src/main/java/com/github/javaparser/ast/CompilationUnit.java [hunk 3]
```diff
@@ -62,25 +58,25 @@
...
 
-    public CompilationUnit(Range range, PackageDeclaration pakage, List<ImportDeclaration> imports,
-                           List<TypeDeclaration<?>> types) {
+    public CompilationUnit(Range range, PackageDeclaration pakage, List<ImportDeclaration> importsList,
+                           List<TypeDeclaration<?>> typesList) {
         super(range);
         setPackage(pakage);
-        setImports(imports);
-        setTypes(types);
+        setImportsList(importsList);
```
Comment (matozoid) [inline javaparser-core/src/main/java/com/github/javaparser/ast/CompilationUnit.java:78]: We could have had a discussion whether we should use "imports" or "importList" - but not "importsList" please.

File: javaparser-core/src/main/java/com/github/javaparser/ast/expr/MethodCallExpr.java [hunk 4]
```diff
@@ -93,9 +93,9 @@ public MethodCallExpr addArgument(Expression arg) {
 		v.visit(this, arg);
 	}
 
-	public List<Expression> getArgs() {
-        args = ensureNotNull(args);
-        return args;
+	public List<Expression> getArgsList() {
```
Comment (matozoid) [inline javaparser-core/src/main/java/com/github/javaparser/ast/expr/MethodCallExpr.java:96]: As noted in the issue, we need to be consistent in args and arguments usage. This is not

File: pom.xml [hunk 5]
```diff
@@ -128,7 +128,7 @@
 		<maven-install-plugin.version>2.5.2</maven-install-plugin.version>
 		<maven-jar-plugin.version>3.0.2</maven-jar-plugin.version>
 		<maven-release-plugin.version>2.5.3</maven-release-plugin.version>
-        <maven-resources-plugin.version>3.0.1</maven-resources-plugin.version>
+        <maven-resourcesList-plugin.version>3.0.1</maven-resourcesList-plugin.version>
```
Comment (ftomassetti) [inline pom.xml:131]: this should have not been changed

File: javaparser-testing/src/test/resources/com/github/javaparser/bdd/parsing_scenarios.story [hunk 6]
```diff
@@ -414,7 +414,7 @@ Scenario: Type Arguments are not specified
 Given a CompilationUnit
 When the following source is parsed:
 class A {
-    List args = new ArrayList();
+    List argsList = new ArrayList();
```
Comment (ftomassetti) [inline javaparser-testing/src/test/resources/com/github/javaparser/bdd/parsing_scenarios.story:417]: Why did you change this?

File: javaparser-core/src/main/javacc/java_1_8.jj [hunk 7]
```diff
@@ -1537,7 +1537,7 @@ BodyDeclaration<?> ClassOrInterfaceBodyDeclaration(boolean isInterface):
 	  {
 	    if(!isInterface)
 	    {
-	      addProblem("A class cannot have default members");
+	      addProblem("A class cannot have default membersList");
```
Comment (matozoid) [inline javaparser-core/src/main/javacc/java_1_8.jj:1540]: Oh come on! Would you want mistakes like this to be in a product you use?

**Comment by matozoid** [comment]
Please, please stop trying to fix this! You are not helping!

**Comment by ftomassetti** [comment]
@rosrossito thank you for your contribution and your enthusiasm. We appreciate it. However I think this pull request cannot be accepted because there are many issues with it.

I suspect you are not very familiar with git and github pull-requests: this is ok, we all started at some time in the past.

In this case there are many changes which are simply wrong and other issues like the fact your branch was not up to date with the changes occurred in the master branch of the original project in the last weeks. I suggest you spend some time looking into git and github and maybe start working on an easier task. Unfortunately we have very much limited time and we are not able to help you and guide you as you would deserve.

I am sorry we have to reject your contribution but I hope it is just the starting point for you and next time things will go better. We all always learning after all.

**Comment by rosrossito** [comment]
Ok. I understand. Thank you.

2016-11-14 22:28 GMT+02:00 Federico Tomassetti notifications@github.com:
